table/tables/tables.go

```diff
@@ -534,6 +534,10 @@ func (t *Table) IterRecords(ctx context.Context, startKey string, cols []*column
 	}
 	defer it.Close()
 
+	if !it.Valid() {
```

Comment:
This check works if and only if the data keys are in the bottom of whole table. But I didn't seem any guarantee about it.